test/spec-testing.js

```diff
@@ -0,0 +1,69 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/loopback-next-hello-world
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+'use strict';
+
+const Dredd = require('dredd');
+const expect = require('@loopback/testlab').expect;
+const HelloWorldApp = require('../lib/application').HelloWorldApp;
+const http = require('http');
+
+const port = 3030; // use this port for Dredd testing only
+const localhost = 'http://localhost:' + port;
+const dreddConfig = {
+  server: localhost + '/', // base path to the end points
+  options: {
+    level: 'verbose',
+    silent: true, // false for helpful debugging info
+    path: [localhost + '/openapi.json'], // to download apiSpec from the service
+  }
+};
+
+describe('Api Spec End Points', () => {
+  let server;
+  let error;
+  let ctFailures;
+  beforeEach(done => {
+    initEnvironment(dreddConfig, (dredd) => {
+      testApi(dredd, done);
+    });
+  });
+  afterEach(done => {
+    server.close();
+    done();
```

Comment:
No need to use `done` since this function is synchronous.

```
afterEach(() => {
  server.close();
});
```

See http://loopback.io/doc/en/contrib/style-guide.html, the hooks should refer to a named function defined at the bottom.